frontend/src/features/collections/collections-add.ts

```diff
@@ -234,47 +203,50 @@ export class CollectionsAdd extends BtrixElement {
   }
 
   private renderSearchResults() {
-    return this.searchTask.render({
-      pending: () => html`
+    if (!this.searchOrg.collections) {
+      html`
         <sl-menu-item slot="menu-item" disabled>
           <sl-spinner></sl-spinner>
         </sl-menu-item>
-      `,
-      complete: (fuse) => {
-        if (!this.hasSearchStr) {
-          return html`
-            <sl-menu-item slot="menu-item" disabled>
-              ${msg("Start typing to search Collections.")}
-            </sl-menu-item>
-          `;
-        }
-
-        const results = fuse
-          ?.search(this.searchByValue)
-          // Filter out items that have been selected
-          .filter(({ item }) => !this.nameSearchMap.get(item))
-          // Show first few results
-          .slice(0, 5);
-
-        if (!results?.length) {
-          return html`
-            <sl-menu-item slot="menu-item" disabled>
-              ${msg("No matching Collections found.")}
-            </sl-menu-item>
-          `;
-        }
+      `;
+    }
+
+    if (!this.hasSearchStr) {
+      return html`
+        <sl-menu-item slot="menu-item" disabled>
+          ${msg("Start typing to search Collections.")}
+        </sl-menu-item>
+      `;
+    }
+
+    const results = this.searchOrg.collections
+      ?.search(this.searchByValue)
```

Comment:
Small thing, but we might want to put a limit on the results returned from fuse for perf reasons. Conservatively, we could probably do 5 + the number of items that have been selected?
```suggestion
      ?.search(this.searchByValue, {
        limit: 5 + this.nameSearchMap.keys.length,
      })
```

Reply:
Good call on limit, addressed the selected items by enabling extended search patterns: https://github.com/webrecorder/browsertrix/pull/2880/commits/077bb977831485cb505aa97ab49ec0e9eafcbe29

Reply:
This is a clever approach, but there are some edge cases that break it. For example, if a collection's name starts with `"` then it will be the only search result that ever shows up if it's selected as an auto-add collection for a workflow:

https://github.com/user-attachments/assets/e9ae7ff2-e983-4a21-8a60-e6f13c808310

I don't know for sure how often this'll come up, but it doesn't seem entirely unfeasible.

Reply:
Hmm yeah I can't think of a user realistically having a collection named `" <something>` ~~since it's the space after the double quote that breaks search, not just starting with double quotes.~~ Quotes in the middle of the name break as well. Looking to see if there's a quick fix.